oak/server/rust/src/asylo_alloc.rs

```diff
@@ -0,0 +1,34 @@
+use core::alloc::{GlobalAlloc, Layout};
+
+extern "C" {
+    // See https://www.sourceware.org/newlib/libc.html#malloc.
```

Comment:
nit: Rather than just including references to other docs, could we include a quick summary comment to indicate what's going on?  That saves an extra click and (more importantly) protects us against link rot in the future.

(Passim)

Reply:
Good point, I'll try to do that more, thanks.